jenkins/release-manifest-commit-lock/release-manifest-commit-lock.jenkinsfile

```diff
@@ -47,7 +47,7 @@ pipeline {
         )
         string(
             name: 'COMPONENTS',
-            description: 'The selected components from the manifest to update the commit ID',
+            description: 'The selected components from the manifest to update the commit ID. Use comma seperate values, example OpenSearch,ml-commons,cross-cluster-replication',
```

Comment:
it would be better it follows the same convention as `COMPONENTS` fields in other jobs, to have space separated values.

Reply:
Yup I agree but with groovy separating out as comma seems to straight froward than what we do with python (the convention for `COMPONENTS`), tried to avoid lot of shell in groovy.